src/server/game/Entities/GameObject/GameObject.h

```diff
@@ -318,7 +328,7 @@ class TC_GAME_API GameObject : public WorldObject, public GridObject<GameObject>
         GameObjectValue m_goValue;
 
         int64 m_packedRotation;
-        QuaternionData m_worldRotation;
+        G3D::Quat m_localRotation;
```

Comment:
This must not be changed - no G3D header should ever be included in any of core headers (compile time reasons)

Reply:
Can I use them in `.cpp` file?